de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/cells/SubjectNameCellDefaultAppearance.java

```diff
@@ -42,14 +45,21 @@ public SubjectNameCellDefaultAppearance(IplantResources iplantResources, Templat
     }
 
     @Override
-    public void render(SafeHtmlBuilder safeHtmlBuilder, Subject subject, String debugID) {
+    public void render(SafeHtmlBuilder safeHtmlBuilder, Subject subject, String debugID, boolean clickableGroupName) {
         String subjectName = subject.getSubjectDisplayName();
         if (subject.getSourceId().equals(Subject.GROUP_IDENTIFIER)) {
-            safeHtmlBuilder.append(templates.group(iplantResources.list().getSafeUri(),
-                                                   SubjectNameCell.SubjectNameCellAppearance.CLICKABLE_ELEMENT_NAME,
-                                                   subjectName,
-                                                   defaultStyle,
```

Comment:
Rather than creating a new `XTemplate` interface method, you could just update this line to `clickableGroupName ? defaultStyle : ""`?

Setting the HTML attribute `class=''` will have the same effect as not adding the `class` attribute at all.